index.html

```diff
@@ -1804,6 +1805,7 @@ <h2>Security Vocabulary Definitions</h2>
           <code>nosec</code> (i.e., <code>"scheme":
           "nosec"</code>), indicating there is no authentication or
           other mechanism required to access the resource.</p></section>
+<section><h3><code>AutoSecurityScheme</code></h3><p>An auto authentication security configuration identified by the term <code>auto</code> (i.e., <code>"scheme": "auto"</code>). This scheme can be used to indicate that the security parameters are going to be negotiated by the underlying protocols at runtime. This may require consumers to prompt users for credentials, e. g. if a username and a password are required for authorized access to a resource.</p></section>
```

Comment:
I would remove the "This may prompt consumers..." etc. sentence.  

Credentials for "auto" should work the same way as in the other schemes, and we in general don't require consumers to issue prompts to users, since they may not even have UIs.  Generally credentials need to be pre-provisioned to consumers but that is a whole other discussion (let's not go down that rabbit hole here).